pkl-commons-cli/src/test/kotlin/org/pkl/commons/cli/BaseCommandTest.kt

```diff
@@ -45,10 +45,10 @@ class BaseCommandTest {
 
   @Test
   fun `external properties without value default to 'true'`() {
-    cmd.parse(arrayOf("-p", "flag1", "-p", "flag2", "-p", "FOO=bar"))
+    cmd.parse(arrayOf("-p", "flag1", "-p", "flag2=", "-p", "FOO=bar"))
```

Comment:
The `flag1` and `flag2` test cases becoming overlapped should have been a hint before ;)